cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1427,7 +1427,7 @@ endif()
 if(vdt OR builtin_vdt)
   if(NOT builtin_vdt)
     message(STATUS "Looking for VDT")
-    find_package(Vdt)
+    find_package(Vdt 0.4)
```

Comment:
Could you add a note to the release notes that this patch release requires at least Vdt 0.4? I think it's nicer to call out new version requirements for patch releases.

Reply:
This was already the case before, since TMVA needs tanh.h, but Vdt didn't export a version, so we couldn't require it. I updated the FindVdt.cmake to guess the version based on the presence of tanh.h so we could ensure it's enough. But sure, I can add to the release notes too.